src/frontends/pytorch/src/op/quantize.cpp

```diff
@@ -69,19 +70,70 @@ OutputVector translate_quantize_per_channel_fx(const NodeContext& context) {
 }
 
 OutputVector translate_fake_quantize_per_tensor_affine_fx(const NodeContext& context) {
-    num_inputs_check(context, 6, 6);
-    auto out = translate_quantize_per_tensor(context);
-    auto axis_0 = context.mark_node(v0::Constant::create(element::i32, Shape{}, {0}));
+    num_inputs_check(context, 5, 6);
```

Comment:
is it possible to provide signature with argument names for better understanding expected arguments and their order?